guides/v2.2/frontend-dev-guide/theme-best-practice.md

```diff
@@ -13,7 +13,7 @@ These approaches give higher chances to avoid conflicts and issues with your the
 
 ## Theme development best practice
 
-1. When [inheriting]({{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html) from a default Magento theme, rather extend, than override the default styles: whenever possible, put your customizations in the `_extend.less` or `_theme.less` file, instead of overriding a `.less` file from a parent theme. 
+1. When [inheriting]({{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html) from a default Magento theme, extend rather than override the default styles, whenever possible. Put your customizations in the `_extend.less` or `_theme.less` file, instead of overriding a `.less` file from a parent theme. 
```

Comment:
```suggestion
1. When [inheriting]({{ page.baseurl }}/frontend-dev-guide/themes/theme-inherit.html) from a default Magento theme, extend the default styles instead of overriding them. Whenever possible, put your customizations in the `_extend.less` or `_theme.less` file, instead of overriding a `.less` file from a parent theme. 
```

Reply:
Apply this edit to all versions/pages as applicable and resubmit.

Reply:
This change has not been applied.